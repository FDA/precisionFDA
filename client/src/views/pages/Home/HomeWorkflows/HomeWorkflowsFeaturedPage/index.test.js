import React from 'react'
import { shallow } from 'enzyme'

import { HomeWorkflowsFeaturedPage } from './index'


describe('HomeWorkflowsFeaturedPage test', () => {
  it('should render', () => {
    const component = shallow(<HomeWorkflowsFeaturedPage />)

    expect(component).toMatchSnapshot()
  })
})


