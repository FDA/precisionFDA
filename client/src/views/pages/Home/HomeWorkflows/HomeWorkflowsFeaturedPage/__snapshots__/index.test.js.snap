// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeWorkflowsFeaturedPage test should render 1`] = `
<HomeLayout>
  <div
    className="home-page-layout__header-row"
  >
    <div
      className="home-page-layout__actions--right"
    >
      <Connect(ActionsDropdown)
        file={Array []}
        makeFeatured={[Function]}
        page="featured"
      />
    </div>
  </div>
  <div
    className="pfda-padded-t20"
  >
    <Connect(HomeWorkflowsFeaturedTable)
      handleFilterValue={[Function]}
      workflows={Array []}
    />
  </div>
</HomeLayout>
`;
